sample/SqliteSample/SqliteSample.csproj

```diff
@@ -1,9 +1,10 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <SqliteVersion Condition="'$(SqliteVersion)'==''">5.0.8</SqliteVersion>
+    <SqliteVersion Condition="'$(SqliteVersion)'==''">7.0.2</SqliteVersion>
     <OutputType>Exe</OutputType>
     <TargetFrameworks>net461;netcoreapp3.1;net5.0;net6.0;net7.0</TargetFrameworks>
+    <PlatformTarget>x64</PlatformTarget>

```

Comment:
Curious why this was necessary to add?

Reply:
The sqlite sample app was failing when trying to load its native library (an issue with an incorrect path). Forcing it to X64 helped and since our profiler only supports X64 I figured this to be an acceptable workaround.